test/e2e/integration/005-splitting-merging-paragraph-blocks.js

```diff
@@ -0,0 +1,24 @@
+describe( 'Splitting and merging paragraph blocks', () => {
+	before( () => {
+		cy.newPost();
+	} );
+
+	it( 'Should split and merge paragraph blocks using Enter and Backspace', () => {
```

Comment:
This test looks like it is testing if `enter` creates a new paragraph and `backspace` removes an empty paragraph. Am I missing something?

Reply:
I had assumed this was the same functionality that's behind splitting and merging.  I was trying to type in the blocks first, but Cypress is absolutely awful when it comes to moving cursors while typing.  I will gladly spend some more time trying to add some text in order to make sure it is also splitting and merging text correctly.

Reply:
Yes, Cypress has some improvements queued in their pipeline. See #4089 and https://github.com/cypress-io/cypress/issues/1108.

Reply:
I know what you meant. When there is a paragraph with text and you type `enter` in the middle. Then move the cursor to the start of next paragraph and type `backspace`. Yes, this is splitting and merging. I think it is specific to the paragraph and a few text-like blocks like heading or list. 

Let's rename to `005-splitting-merging-blocks.js` and add some text inside the blocks.